src/thor/trace_attributes_action.cc

```diff
@@ -154,6 +154,10 @@ std::string thor_worker_t::trace_attributes(valhalla_request_t& request) {
     throw valhalla_exception_t{442};
   }
 
+  if (std::get<kTripLegIndex>(map_match_results.at(0)).empty()) {
+    throw valhalla_exception_t{442};
```

Comment:
this is redundant with the for loop below

Reply:
actually im wrong. but this seems to be papering over something larger. in what circumstance would this ever happen?